tests/garage/torch/modules/test_mlp_module.py

```diff
@@ -138,4 +138,5 @@ def test_mlp_with_learnable_non_linear_function(self):
         output.sum().backward()
 
         for tt in module.parameters():
-            assert torch.all(torch.ne(tt.grad, 0))
+            if tt.grad is not None:
```

Comment:
can you explain why this branch is needed?

conditionals in test cases usually indicated something fishy is going on

Reply:
There's supposed to be no changes in this file. I forgot to delete it.